components/View/style.js

```diff
@@ -37,16 +34,16 @@ const content = (
     top: isFullscreen ? 0 : `calc(${navHeight}px + var(--safe-area-inset-top))`,
     display: 'flex',
     flexDirection: 'column',
-    paddingBottom: 'calc(var(--tabbar-height) + var(--safe-area-inset-bottom))',
     bottom: 0,
-    ...considerPaddingTop && {
-      marginBottom: 'calc(var(--tabbar-height) + var(--safe-area-inset-bottom))',
-      bottom: -24,
-    },
   }).toString();
 };
 
+const padding = css({
```

Comment:
There is an issue when the product list widget is the last entry within a widget configuration. It has a visible gap blow the `load more` button which is caused by a margin. Replacing this with a padding will help
![img_0121](https://user-images.githubusercontent.com/13087792/43011423-af160d1e-8c43-11e8-90a3-3f6c8971be26.PNG)